examples/escrow/1.0.0.json

```diff
@@ -52,10 +53,10 @@
     },
     "Escrow": {
       "runtime_bytecode": {
-        "bytecode": "0x606060405260e060020a600035046366d003ac811461003457806367e404ce1461005d57806369d8957514610086575b610000565b3461000057610041610095565b60408051600160a060020a039092168252519081900360200190f35b34610000576100416100a4565b60408051600160a060020a039092168252519081900360200190f35b34610000576100936100b3565b005b600154600160a060020a031681565b600054600160a060020a031681565b60005433600160a060020a039081169116141561014857600154604080516000602091820152815160e260020a6324d048c7028152600160a060020a03938416600482015230909316316024840152905173000000000000000000000000000000000000000092639341231c926044808301939192829003018186803b156100005760325a03f41561000057506101e2915050565b60015433600160a060020a039081169116141561002f576000805460408051602090810193909352805160e260020a6324d048c7028152600160a060020a039283166004820152309092163160248301525173000000000000000000000000000000000000000092639341231c9260448082019391829003018186803b156100005760325a03f41561000057506101e2915050565b610000565b5b5b56",
+        "bytecode": "0x6080604052600436106100565763ffffffff7c010000000000000000000000000000000000000000000000000000000060003504166366d003ac811461005b57806367e404ce1461008c57806369d89575146100a1575b600080fd5b34801561006757600080fd5b506100706100b8565b60408051600160a060020a039092168252519081900360200190f35b34801561009857600080fd5b506100706100c7565b3480156100ad57600080fd5b506100b66100d6565b005b600154600160a060020a031681565b600054600160a060020a031681565b600054600160a060020a031633141561019857600154604080517f9341231c000000000000000000000000000000000000000000000000000000008152600160a060020a039092166004830152303160248301525173000000000000000000000000000000000000000091639341231c916044808301926020929190829003018186803b15801561016657600080fd5b505af415801561017a573d6000803e3d6000fd5b505050506040513d602081101561019057600080fd5b506102289050565b600154600160a060020a031633141561005657600054604080517f9341231c000000000000000000000000000000000000000000000000000000008152600160a060020a039092166004830152303160248301525173000000000000000000000000000000000000000091639341231c916044808301926020929190829003018186803b15801561016657600080fd5b5600a165627a7a723058201766d3411ff91d047cf900369478c682a497a6e560cd1b2fe4d9f2d6fe13b4210029",
         "link_references": [
           {
-            "offsets": [262, 412],
+            "offsets": [301, 495],
```

Comment:
What are those extra `0`s blocks doing?

Reply:
Hard for me to tell - my opcode game isn't solid. But that's the `runtime_bytecode` output when remix compiles the contract - and they're also there if you check the runtime code on Etherscan. I don't think they have anything to do with our link references... 🤷‍♂️